app/BUILD.bazel

```diff
@@ -246,8 +246,6 @@ VIEW_MODELS = [
     "src/main/java/org/oppia/android/app/topic/revision/TopicRevisionViewModel.kt",
     "src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardViewModel.kt",
     "src/main/java/org/oppia/android/app/topic/TopicViewModel.kt",
-    "src/main/java/org/oppia/android/app/viewmodel/ObservableArrayList.kt",
-    "src/main/java/org/oppia/android/app/viewmodel/ObservableViewModel.kt",
```

Comment:
@BenHenning for https://github.com/oppia/oppia-android/pull/2648#discussion_r578125730 I've removed these from the VIEW_MODELS list.

EXCLUDED_APP_LIB_FILES includes VIEWMODELS list
https://github.com/oppia/oppia-android/blob/5c5328ab0dbca246a6c5f90e109fe093c17bd338/app/BUILD.bazel#L407

and the EXCLUDED_APP_LIB_FILES list is excluded from the glob
https://github.com/oppia/oppia-android/blob/5c5328ab0dbca246a6c5f90e109fe093c17bd338/app/BUILD.bazel#L411-L415